src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -54,10 +54,12 @@ public static IEnumerable<RegexPattern> HighConfidenceSecurityModelsIterator()
         new Unclassified32ByteBase64String(),
         new Unclassified64ByteBase64String(),
         new Unclassified16ByteHexadecimalString(),
+        // new UrlCredentials()
```

Comment:
>// new UrlCredentials() [](http://example.com/codeflow?start=8&length=23)

remove?

Reply:
This needs to be added in. It's commented since the rule is broken.

Reply:
Added comment to GH issue